docs/api_reference/index.rst

```diff
@@ -7,6 +7,7 @@ API Reference
    :maxdepth: 1
 
    fairlearn.metrics
+   fairlearn.metrics_plots
```

Comment:
I'm in favor of having plotting be always in the modules that they support. This is consistent with how [sklearn handles this](https://scikit-learn.org/stable/modules/classes.html#id3) and also how we have done it with `fairlearn.postprocessing.plot_threshold_optimizer` (I think we had a conversation about it back then).

Reply:
I second this

Reply:
Awesome, that works for me!